### PR TITLE
Support to add command line info into bitstream using sei messages and vui params modification into to string

### PR DESCRIPTION
**Pull request by mamthan**

### CODE REVIEW

File: src_base/xeve_eco.c [hunk 1]
```diff
@@ -340,11 +340,71 @@ int xeve_eco_signature(XEVE_CTX * ctx, XEVE_BSW * bs)
     return XEVE_OK;
 }
 
+void write_sei_userdata_unregistered(XEVE_SEI_PAYLOAD * sei_userdata, XEVE_BSW * bs)
```
Comment (kpchoi) [inline src_base/xeve_eco.c:343]: if this function is gloabl function, "xeve_" prefix is required at the begin of function name.
But, if it is not global function, it should be declared "static" function.

**Review by kpchoi — COMMENTED** [review]
Please find my comments.

File: src_base/xeve_eco.c (hunk 1 above)
Comment (mamthan) [reply src_base/xeve_eco.c]: It is not a golabal function, have modified it as static function.

**Review by kpchoi — APPROVED** [review]
LGTM

**Pull request merged**